CONTINUITY.md

```
@@ -1,67 +1,61 @@
-bench update limpio en producción (sin crash de migrate).
+Departamentos grupo visibles en Configuración CFDI Recibidos y en modal
+"Asignar Departamentos" en lista de CFDI Recibidos.
 
 ---
 
 ## Estado actual
 
-### Ya completado
-- ✅ acg-v16.dev configurado completamente (CoA, fiscal, items, La Comer)
-- ✅ PR abierto con 4 commits
+### Ya cerrado
```

markdownlint-cli2 (0.22.1)

[warning] Headings should be surrounded by blank lines
Expected: 1; Actual: 0; Below

(MD022, blanks-around-headings)